django_project/frontend/src/app/index.jsx

```diff
@@ -71,7 +72,9 @@ export function render(App, store) {
   root.render(
     <Provider store={store}>
       <ConfirmDialogProvider>
-        <App/>
+        <BrowserRouter>
```

Comment:
Hi @zamuzakki 
We don't need this, just access the url when the table is rendered